ansible_rulebook/schema/ruleset_schema.json

```diff
@@ -106,18 +115,49 @@
                         "post_events": { "type": "boolean" },
                         "set_facts": { "type": "boolean" },
                         "verbosity": {"type": "integer" },
-                        "var_root": {"type": "string" },
+                        "var_root": {"type": ["string","object"] },
+                        "json_mode": { "type": "boolean" },
+                        "retry": { "type": "boolean" },
+                        "retries": { "type": "integer" },
+                        "delay": { "type": "number" }
+                    },
+                    "required": [
+                        "name"
+                    ],
+                    "additionalProperties": true
```

Comment:
This field should be `false` (run module has it too)
Otherwise an action like:
```
      action:
        run_playbook:
          name: playbooks/hello.yml
          action: "wat"
```
Is not checked and throws the following exception:
```
ERROR:ansible_rulebook.engine:Error calling run_playbook
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/ansible_rulebook/engine.py", line 451, in execute
    result = await run_playbook(**action_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/ansible_rulebook/builtin.py", line 224, in run_playbook
    temp, playbook_name = await pre_process_runner(
                                ^^^^^^^^^^^^^^^^^^^
TypeError: pre_process_runner() got multiple values for argument 'action'
```

Reply:
@Alex-Izquierdo In the run module we have an attribute called **module_args** which allows you to set all the module specific args in a single object. And the additional properties is set to false for run_module.
For run_playbook should we add a **playbook_args** and then we can set additional properties to false.

Reply:
ok then